src/server/items/models.js

```diff
@@ -0,0 +1,34 @@
+import db from '../database';
+import {KeyUtils, STATE, DEFAULT_PRECISON} from '../key-utils';
+
+export default class ItemManager {
```

Comment:
I think this belongs to key-utils.js.
this state change function can be shared the other entities(images, reports, etc ...)  later.